### ISSUE TITLE
[Resizeable webclent text] Allow text within the web client to be resized and recolored independent of browser settings.

### ISSUE THREAD

**Issue by Cantir2068** [evennia/evennia#2218]
#### Describe the solution you'd like
An option within the web portal's options settings  to allow font size and type manipulation that does not rely on client browser don't settings, or zoom level.

Additional color settings would benefit for the sake of allowing end users to adjust what contrasts they are comfortable with for the default presentation of text independent of any softcore settings for indavidual tags within game, as some prefer full white on black,others prefer a light grey, beige, and others prefer a light green for aesthetics.

#### Describe alternatives you've considered
A clear and concise description of any alternative solutions
Normally simply bumping zoom or font up would be feasable, however considering the portal window to evennia itself is a separate element to the rest of the page it is in, adjusting global text values within client browser could cause unforseen formatting and readability issues based on client desktop resolution, fonts, windowed vs fullscreen, and so forth.

#### Additional context

With best correction I have 20/300 vision, have worked with text enlargement adaptive technology for close to thirty years,and have been within one mud/mush/etc community or other since roughly the mid 90s using everything from windows 95 terminal emulator, Linux terminal window, putty, mushclient, tintin, and potato.

I don't have all the answers, but a higher than average percentage of MU* players have vision issues of one stripe or another based on how friendly raw text is for visual immparement.
With best correction I have 20/³⁰

**Comment by friarzen** [comment]
Font and Font Size controls were recently added to the settings tab that can be opened via the "Cog" Icon on the top left of "Main" pane.

**Comment by Griatch** [comment]
@friarzen That's a good point, I had forgotten about those. @Cantir2068 You can set your own fonts if you open the settings (cogwheel) of the client. Is this enough to satisfy this request? If not, could you elaborate on what is missing?

**Comment by Vikka** [comment]
> * Colors can be changed through CSS in your browser, in principle, but yeah, it's not as easy as clicking an option, that's for sure.

What do you think about this: An option panel that let you choose the different color used in the webclient ?

The goal is to save the settings and to make the color change easier than the CSS alternative.

**Comment by Griatch** [comment]
@Vikka Yes, that would of course be possible; there are a few issues before one could do something like that though;
- Not all colors are controlled via easy CSS classes. If I'm not mistaken, some refactoring of the CSS structure needs to happen to make things easier to modify (without anything getting hard-coded/set directly in JS). @friarzen probably knows more about the state of this though. 
- Today we don't have a mechanism for storing a user's settings across sessions/logins. A more fundamental change to the webclient/server infrastructure needs to be in place before we can do that.

**Comment by Cantir2068** [comment]
@Griatch I would think having text and color in the client cogwheel menus would be suitible, but with no way to save options it will become a bit of a routine chore. However I view font/default color to be acceptable, yes.

**Comment by Griatch** [comment]
@friarzen @2224 solves persistence of layout; there is also the ability to change the font and size as well, but it appears like while the font type and size is saved across reload now, said font/size doesn't actually apply to the look of the webclient :) Reloading the browser page resets back to the default font/size even though the option still lists your custom font/size. I think with the fix of that, this feature-request could be closed as well.
